client/src/Components/Tendency/Tendency.jsx

```diff
@@ -0,0 +1,130 @@
+            </div>
+            <div className="title-wrap-item">
+              <div className="wrap-item" id="dropdownMenuButton" data-toggle="dropdown">
+                <small className="dropdown-toggle">Mới nhất <i className="fas fa-angle-down pr-2" /></small>
+              </div>
+              <ul className="wrap-drop dropdown-menu" aria-labelledby="dropdownMenuButton">
+                <li className="wrap-drop-item">
+                  <a href="/">Mới nhất</a>
+                </li>
+                <li className="wrap-drop-item">
+                  <a href="/">Tin tiêu biểu</a>
+                </li>
+                <li className="wrap-drop-item">
+                  <a href="/">Phổ biến nhất</a>
+                </li>
+                <li className="wrap-drop-item">
+                  <a href="/">Xem nhiều theo tuần</a>
+                </li>
+                <li className="wrap-drop-item">
+                  <a href="/">Ngẫu nhiên</a>
+                </li>
+              </ul>
+            </div>
+          </div>
+        </div>
+        {/* section1 mới */}
+        <section className="new">
+          <div className="container">
+            <div className="row Content-new">
+              {
+                items.map((value,key) => {
+                  if(value.id === 8){
+                    return (
+                      <div className="col-md-6 i1" key={key}>
+                        <div className="img-1">
+                          <a href={`/Detail/${value.id}`}>
+                            <img src={value.img} alt={value.name} />
+                          </a>
+                        </div> 
+                        <div className="Title-1">
+                          <small>Xu Hướng</small>
+                          <h3>{value.name} </h3>
+                          <span>{value.date}</span>
+                        </div>
+                      </div>
+                    )
+                  }
+                  else{
+                    return null;
+                  }
+                })
+              }
+              <div className="col-md-6 i2">
+                <div className="card-2">
+                 {
+                   items.map((value,key) => {
+                     if(value.id >= 7){
+                       return <TendencyListNewItem key={key} Id={key} Name={value.name} Image={value.img}/>
+                     }
+                     else{
+                      return null;
+                     }
+                   })
+                 }                    
+                </div>
+              </div>
+            </div>
+          </div>
+        </section>
+        {/* end section1 mới */}
+        {/* Tin tức */}
+        <section className="xuhuong">
+          <div className="container">
+            {
+             items.map((value,key) => 
+              <TendencyListItem key={key} Id={key} Name={value.name} Content={value.content} Date={value.date} Image={value.img}/>
```

Comment:
em tìm hiểu về https://viblo.asia/p/three-dots-da-thay-doi-javascript-nhu-the-nao-QpmlenBr5rd
rồi sửa lại chỗ này cho anh